feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-crypto/main/module.xml

```diff
@@ -38,7 +38,7 @@
         <module name="org.apache.james.mime4j"/>
         <module name="javax.servlet.api"/>
         <module name="javax.ws.rs.api"/>
-        <module name="org.bouncycastle"/>
+        <module name="org.bouncycastle" export="true"/>
```

Comment:
I agree with Stuart, bouncycastle should never be exported by default, as we already had problems in the past because of this.
We should find out what module really needs it and add dependencies there.